src/main/java/uk/gov/justice/hmpps/casenotes/dto/NomisCaseNote.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.hmpps.casenotes.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@ApiModel(description = "Case Note")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@Data
+public class NomisCaseNote {
+
+    @ApiModelProperty(required = true, value = "Case Note Id (unique)", position = 1, example = "12311312")
+    @NotNull
+    private Long caseNoteId;
```

Comment:
Noticed that `CaseNoteController` now references `caseNoteIdentifier` so should this here too?

Reply:
Well NomisCaseNote is an elite2 json dataset which is named differently - this is transformed into a CaseNote dto afterwards.

Reply:
oops agreed